.github/workflows/ci.yml

```diff
@@ -43,6 +43,8 @@ jobs:
             python -m pip install --upgrade pip
             pip install -e py
             pip install pylint
+            pip install numpy
+            pip install scipy
```

Comment:
These requirements should be added to stabilizer's `setup.py` file. Otherwise, we have a broken package.